src/main/java/dukeApp/files/Deadline.java

```diff
@@ -0,0 +1,94 @@
+package dukeApp.files;
+
+public class Deadline extends Task {
+    protected String word;
+    protected String date;
+    protected String time;
+    protected String dateLine;
+    protected String originalDate;
+
+    public Deadline(String s) {
```

Comment:
Here, you assume that the string is always valid. If the string is not valid, then many of your steps can cause an error, e.g. arrayindexoutofbounds. Hence it would be better to use a try catch block to handle invalid inputs